pkg/printers/internalversion/describe.go

```diff
@@ -837,6 +839,34 @@ func printConfigMapVolumeSource(configMap *api.ConfigMapVolumeSource, w PrefixWr
 		configMap.Name, optional)
 }
 
+func printProjectedVolumeSource(projected *api.ProjectedVolumeSource, w PrefixWriter) {
+	infos := []string{}
+	for _, source := range projected.Sources {
+		if source.Secret != nil {
+			infos = append(infos, fmt.Sprintf("       - Secret:\n"+
```

Comment:
Indent by 4 spaces here

Reply:
Also, to keep consistency with other volume describers, I'd be in favor of removing the `-` at the beginning of this section, and just squashing it down so it reads like:

```
    SecretName: ...
    SecretOptional: ...
    TokenExpirationSeconds:
    ...
```